Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+
+<html <head>
+
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+<title>GD2 PR1</title>
+
+<link rel="stylesheet" <link rel="stylesheet" href="Styles.css">
+
+</head>
+
+<body>
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+
+    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+        intelligent.</h2>
+
+
+    <p class="article-meta">By Carl Zimmer</p>
+
+    <p class="article-meta">Nov. 30, 2018</p>
+
+    <figure>
+
+        <img src="Images/merlin_147494628_a788760f-4209-40db-a0be-1305c3a661a6-superJumbo.jpg" width="1000" height="500"
+            alt="octopus swimming under the sea">
+
+        <figcaption>
+            An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
+            uncertain how the trait evolved.</figcaption>
+
+    </figure>
+
+    <p> To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus
+        pulling
+        two halves of a coconut shell together to hide inside. Later the animal stacks the shells together like
+        nesting
+        bowls — and carts them away.</p>
+
+    <blockquote>“The octopus is carrying these tools around because it has some understanding they may be useful in
+        the
+        future,” said Mr. Amodio</blockquote>
+
+    <p>But his amazement is mixed with puzzlement.</p>
+
+    <p>For decades, researchers have studied how certain animals evolved to be intelligent, amoung them apes,
+        elephants,
+        dolphins and even some birds, such as crows and parrots.</p>
+
+    <p>But all the scientific theories fail when it comes to cephalopods, a group that includes octopuses, squid
+        and cuttlefish. Despite feats of creativity, they
+        <a href=<cite>lack some hallmarks</cite></a>
+        of intelligence seen in other species.
```

Comment:
Missing closing `p` tag here:
```suggestion
        of intelligence seen in other species.</p>
```